pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -193,6 +193,34 @@ var _ = Describe("Pod Eviction Cost", func() {
 })
 
 var _ = Describe("Replace Nodes", func() {
+	It("should ignore nodes without TTLSecondsUntilExpired", func() {
```

Comment:
I had it here since there are some other "Block Deprovisioning Unit Tests" here that are used for replace nodes. I'll move it to the Delete Nodes below.